drv/psc-psu-update/src/main.rs

```diff
@@ -0,0 +1,262 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#![no_std]
+#![no_main]
+
+use drv_i2c_api::*;
+use drv_i2c_devices::mwocp68::{
+    Error as Mwocp68Error, Mwocp68, Mwocp68FirmwareRev, UpdateState,
+};
+use drv_i2c_devices::Validate;
+use ringbuf::*;
+use static_cell::ClaimOnceCell;
+use userlib::*;
+
+use core::ops::Add;
+
+task_slot!(I2C, i2c_driver);
+
+const TIMER_INTERVAL: u64 = 10000;
```

Comment:
```suggestion
const TIMER_INTERVAL: u64 = 10_000;
```
(consider renaming to `TIMER_INTERVAL_MS` as well?)